hash-fetch/src/urlhint.rs

```diff
@@ -85,14 +89,16 @@ pub struct Content {
 	/// MIME type of the content
 	pub mime: Mime,
 	/// Content owner address
-	pub owner: Address,
+	pub owner: Address
```

Comment:
In rust we try to have trailing commas after each line.

Reply:
Can't there be a linter for this?

Reply:
I'd love to have one, we tried using `rustfmt` some time ago, but there were many places were the formatting was a bit weird. We should give it a go again soon though!